src/port1.0/portutil.tcl

```diff
@@ -3469,3 +3469,22 @@ proc _archive_available {} {
     set archive_available_result 0
     return 0
 }
+
+# get the mountpoint providing a given directory
+proc get_mountpoint {target_dir} {
+    file stat ${target_dir} target_stat
+
+    set parentdir ${target_dir}
+
+    while {$parentdir != "/"} {
```

Comment:
```suggestion
    while {$parentdir ne "/"} {
```

In Tcl, "eq" and "ne" perform string comparisons, while "==" and "!=" attempt to perform integer comparisons first and then fall back to string comparisons.